randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -867,14 +895,14 @@ Extra - room_id: [28]
   * L0 Hatch to Nightmare Hub West/Door to Flooded Access
   * Extra - door_idx: (63,)
   > Door to Zebesian Waters
-      Trivial
+      Normal Damage ≥ 250 or Can Kill Tough Beam-Resistant Enemy
```

Comment:
do we want the damage to be there on trickless? You usually have a way in vanilla to fullfill the tough beam resistant template, so feel like this should be gated behind some damage boost beginner trick instead.